code/game/area/areas.dm

```diff
@@ -40,7 +40,15 @@
 	var/static_light = 0
 	var/static_environ
 
-	var/has_gravity = 0
+	var/has_gravity = 0				//Has gravity innately/no matter what. Ignores gravity generator gravity direction.
+	var/list/contents_affected_by_gravity = list()
```

Comment:
Uh, this will almost never be empty, unless you're trying to save init time.
Scratch that, it'll never be empty unless you literally only have walls in somewhere.